src/main/java/com/fasterxml/jackson/databind/ser/AnyGetterWriter.java

```diff
@@ -25,10 +25,14 @@ public class AnyGetterWriter
 
     protected MapSerializer _mapSerializer;
 
+    /**
+     * @since 2.19
+     */
     @SuppressWarnings("unchecked")
-    public AnyGetterWriter(BeanProperty property,
+    public AnyGetterWriter(BeanPropertyWriter parent, BeanProperty property,
             AnnotatedMember accessor, JsonSerializer<?> serializer)
     {
+        super(parent);
```

Comment:
Is this actually safe? Doesn't it copy settings of "parent"?

Reply:
It was intended actually. Copying seemed safer than holding reference to original BeanPropertyWriterpropert propert that is soon to be a any-getter.
I referenced same pattern in `VirtualBeanPropertyWriter`, `UnwrappingBeanPropertyWriter`, like below.
Do you think there would be side effect?

```java

    public UnwrappingBeanPropertyWriter(BeanPropertyWriter base, NameTransformer unwrapper) {
        super(base);
        _nameTransformer = unwrapper;
    }

    protected UnwrappingBeanPropertyWriter(UnwrappingBeanPropertyWriter base, NameTransformer transformer,
            SerializedString name) {
        super(base, name);
        _nameTransformer = transformer;
    }
```